addons/web/controllers/main.py

```diff
@@ -1066,7 +1055,20 @@ def _render_template(self, **d):
             monodb = db_monodb()
             if monodb:
                 d['databases'] = [monodb]
-        return Markup(env.get_template("database_manager.html").render(d))
+
+        templates = {}
+
+        with file_open("web/static/src/public/database_manager.qweb.html", "r") as fd:
+            template = fd.read()
+        with file_open("web/static/src/public/database_manager.master_input.qweb.html", "r") as fd:
+            templates['master_input'] = fd.read()
+        with file_open("web/static/src/public/database_manager.create_form.qweb.html", "r") as fd:
+            templates['create_form'] = fd.read()
+
+        def load(template_name, options):
+            return (html.fragment_fromstring(templates[template_name]), template_name)
+
+        return Markup(QWeb()._render(html.document_fromstring(template), d, load=load))
```

Comment:
Don't think the wrapping is necessary, `Qweb._render` should return `Markup` objects (to verify).